SQL_assignment/sql_assignment1

```diff
@@ -0,0 +1,82 @@
+CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+
+    Create table emp(EmpoyeeID int primary key,firstname varchar(20),lastname varchar(20),department varchar(20),salery int)
+
+
+
+INSERT: Adds three employee records to the table.
+
+    insert into emp values(1,'akash','j','IT',50000)
+    insert into emp values(,2'arun','p','NON-IT',32000)
```

Comment:
will this work ?

Reply:
yes

Reply:
```suggestion
    insert into emp values(2,'arun','p','NON-IT',32000)
```

now only this will work, right

Reply:
yes, it was a mistake while copying the query to text file.